javascript/javascript1/week1/app.js

```diff
@@ -0,0 +1,73 @@
+//STEP 2
+//user freecodecamp fccdc38edef-ba87-461c-96cd-bf07dcfc8098
+
+//STEP 3
+
+/*Age-ify*/
+var yearOfBirth = 1987; 
```

Comment:
in cases like this it is preferred to use `const` since the variable is not changed

Reply:
Yes, you are right.